cadastro_perguntas/app/app/perguntas/admin.py

```diff
@@ -32,41 +53,43 @@ def add_view(self, request, form_url='', extra_context=None):
         except ValidationError as e:
             print(e)
 
-    # def get_changeform_initial_data(self, request):
-    #     get_data = super(PerguntaAdmin, self).get_changeform_initial_data(request)
-    #     get_data['criado_por'] = request.user.id
-    #     return get_data
-
-    def get_queryset(self, request):
-        qs = super().get_queryset(request)
-        if request.user.is_superuser or request.user.has_perm('review_question'):
-            return qs
-        # Filtrar apenas que ainda não tenham sido revisadas
-        return qs.filter(criado_por=request.user)
-
     def formfield_for_foreignkey(self, db_field, request, **kwargs):
         if db_field.name == "criado_por":
             kwargs["initial"] = request.user.id
             kwargs["disabled"] = True
         return super().formfield_for_foreignkey(db_field, request, **kwargs)
 
     def change_view(self, request, object_id, extra_context=None):
-        if request.user.has_perm('review_question'):
-            self.fields.append('revisado_por')
-            self.fields.append('revisado_em')
-            self.readonly_fields.append('revisado_por')
-            self.readonly_fields.append('revisado_em')
-        if request.user.has_perm('publish_question'):
-            self.fields.append('publicado_por')
-            self.fields.append('publicado_em')
-            # self.readonly_fields = ('revisado_por','revisado_em')
+        if request.user.groups.filter(name='revisores').exists():
+            self.fields[6] = ('revisado_status',)
+        if request.user.groups.filter(name='publicadores').exists():
+            self.fields[6] = ('status',)
+        if request.user.groups.filter(name='administradores').exists():
+            self.fields[6] = ('revisado_status', 'status')
+
+        return super().change_view(request, object_id, extra_context)
 
     def save_model(self, request, obj, form, change):
-        # Se quem salvou é PUBLICADOR, então ele ganha o publicado_em e publicado_por
-        obj.user = request.user
-        super().save_model(request, obj, form, change)
+        if not obj.revisado_status:
+            obj.status = False
+
+        if obj.revisado_status:
```

Comment:
- [ ] Lembre de, se possível, como já foi comentado, alterar o texto do botão de salvar para "Revisar" e "Publicar" dependendo da apção do momento https://stackoverflow.com/questions/62622374/how-to-save-model-data-using-custom-button-instead-of-using-save-button-in-djang
- [ ] Também é interessante alterar o idioma do admin para português, pois os usuários internos serão brasileiros https://groups.google.com/g/pug-ce/c/BbceAZQAOIg